lib/chef/knife/config.rb

```diff
@@ -0,0 +1,126 @@
+#
+# Author:: Joshua Timberman <opensource@housepub.org>
```

Comment:
?

Reply:
I started from https://github.com/jtimberman/knife-config/blob/master/lib/chef/knife/config.rb, though there is little of that code left I think :)